code/gradle/context/sources.gradle

```diff
@@ -0,0 +1,8 @@
+sourceSets {
+    main {
+        java {
+            /** Include apollo generated sources in the final JAR **/
+            srcDirs += 'build/generated/source/apollo'
```

Comment:
FYI, this was removed in a recent PR (when we switched to Kotlin models)

Reply:
Is the change reflected on the feature branch? I am unaware of that.

Reply:
No, it's okay for now. But I think we need to sync the feature branch with the main soon.